src/components/utils/tooltip.ts

```diff
@@ -48,3 +46,5 @@ export default class Tooltip extends Module {
     this.lib.onHover(element, content, options);
   }
 }
+
+export default new Tooltip();
```

Comment:
its better to create an instance in a class constructor or in the method where it used, rather than use a singletone